docs/graph_engine/temporal.rst

```diff
@@ -0,0 +1,119 @@
+Temporal graph support
+======================
+
+DeepGNN suports temporal graphs, i.e. graphs with timestamps assigned to each element:
+nodes, edges, or features. Lets explore how to use them.
+
+Node
+----
+
+A new node can be added or removed to a graph.
+A JSON schema for temporal nodes is an extension of `regular<data_spec.html>`_ nodes with two new fields: `created_at`
+and `removed_at` with values recorded in cardinal format. Here is a concrete example:
```

Comment:
I don't think cardinal format is it. Maybe "with integer / float timestamp values."?

Reply:
Updated to UNIX ts.